apps.toml

```diff
@@ -3095,6 +3095,12 @@ state = "notworking"
 subtags = [ "forge" ]
 url = "https://github.com/drfred1981/scm-manager_ynh"
 
+[scovie]
+category = "publishing"
+state = "working"
+subtags = [ "websites" ]
+url = "https://github.com/eldertek/scovie_ynh"
```

Comment:
it will be better if you can transfer the app to the YunoHost app repo

Reply:
Changed to yunohost-apps, but cannot transfer ownership : "You don’t have the permission to create public repositories on YunoHost-Apps"

Reply:
invite send

Reply:
I imported the repository !